spec/acceptance/suites/two_domains/00_rsyslog_forward_spec.rb

```diff
@@ -0,0 +1,183 @@
+require 'spec_helper_acceptance'
+
+test_name 'simp_rsyslog profile'
+
+describe 'simp_rsyslog' do
```

Comment:
simp_beaker_helpers already resets the interfaces, so it may not be necessary here